Bottom: 3e284195875c201602d05798a8c7b6672a5a4fd0
Top:    fac78ffe77c22e0621a021a0a4e1d49994f8a8d7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:38:54 +0100

Refresh of off-to-find-burried-treasure-o

---

diff --git a/src/find.c b/src/find.c
index e4c114d..af9103c 100644
--- a/src/find.c
+++ b/src/find.c
@@ -4028,17 +4028,17 @@ IsPolygonAnIsland (LayerType *layer, PolygonType *polygon)
    */
   TheFlag = FOUNDFLAG | DRCFLAG;
 
-  ResetConnections (False);
+  ResetConnections (false);
 
   /* Let the search stop if we find something we haven't yet seen */
-  drc = True;
-  User = False;
+  drc = true;
+  User = false;
 
   ListStart (POLYGON_TYPE, layer, polygon, polygon);
 
   do
     {
-      any_more = DoIt (False, False);
+      any_more = DoIt (false, false);
 
       /* Check if we got any useful hits */
       connected_count = 0;
@@ -4057,8 +4057,8 @@ IsPolygonAnIsland (LayerType *layer, PolygonType *polygon)
     }
   while (any_more);
 
-  drc = False;
-  ResetConnections (False);
+  drc = false;
+  ResetConnections (false);
   FreeConnectionLookupMemory ();
 
   return (connected_count == 0);
